spec/factories/cities.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :city do
+    city_place_id { "MyString" }
+    name { "MyString" }
```

Comment:
"MyString" is just a placeholder. I suggest:

```suggestion
    city_place_id { SecureRandom.hex(51) }
    name { Faker::Address.city }
```

Reply:
Thank you for showing this @dsinn now I can mimic it in my own PRs